# @ref adasis_v2.h
int64 time_stamp

uint8 type

uint8 cyclic_counter

bool retransmission

uint8 path_index

bool update

uint8 profile_type

bool control_point

uint16 offset

float32 value0

uint16 distance1

float32 value1

uint8 accuracy
